api/types.proto

```diff
@@ -771,6 +783,10 @@ message RootCA {
 	// RootRotation contains the new root cert and key we want to rotate to - if this is nil, we are not in the
 	// middle of a root rotation
 	RootRotation root_rotation = 5;
+
+	// ForceRotationVersion matches the Cluster Spec's CAConfig's ForceRotation counter.
+	// It indicates when the current CA cert and key were generated (or updated).
+	uint64 force_rotation_version = 6;
```

Comment:
Maybe just `RotationVersion`?

Maybe `LastForcedRotation` is better, actually, because that is less likely to be confused with the `Version` field in `Meta`.